src/HtmlPurifier.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We don't use the PSR-2 conventions in our code, we use a CS based on the [MediaWiki CS](https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP) which advocates lots of spaces:

> Put spaces next to parentheses on the inside, except where the parentheses are empty. 

Since adopting PSR-12, which supersedes PSR-2, would mean lots of other changes to our CS, I'd say we continue using spaces until the MediaWiki CS clarifies its handling of `declare` statements.